cli_main.py

```diff
@@ -305,9 +432,9 @@ def main(
 
     # Convert special types (i.e AtomGroups)
     # Ugly that we have to parse again... but currently I have no better idea :(
-    for doc_param in NumpyDocString(analysis_callable.__doc__)["Parameters"]:
-        if "AtomGroup" in doc_param.type:
-            analysis_kwargs[doc_param.name] = u.select_atoms(analysis_kwargs[doc_param.name])
+    for doc_param in parse_docs(analysis_callable)[2]:
+        if "AtomGroup" in doc_param['type']:
+            analysis_kwargs[doc_param['name']] = u.select_atoms(analysis_kwargs[doc_param['type']])
 
     ac = analysis_callable(**analysis_kwargs)
```

Comment:
In this line our code will break since the `analysis_kwargs` dictionary contains the keys `begin`, `end`, `dt`, `verbose` and `func`. I fixed this, but we should maybe clean up the  `analysis_kwargs ` dict. It should only contain the entries for the analysis class... What do you think

Reply:
raised an issue for it #5